File: CS/AngularReportingLocalizationExample/Controllers/WebDocumentViewerController.cs
﻿using DevExpress.Web.Mvc.Controllers;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace AngularReportingLocalizationExample.Controllers
{
    public class WebDocumentViewerController : WebDocumentViewerApiControllerBase
    {
        // GET: WebDocumentViewerApi
        public ActionResult Index()
        {
            return View();
        }
        public override ActionResult Invoke()
        {
            var result = base.Invoke();
            Response.AppendHeader("Access-Control-Allow-Origin", "*");
            return result;
        }
    }
}

File: VB/AngularReportingLocalizationExample/Controllers/WebDocumentViewerController.vb
﻿Imports DevExpress.Web.Mvc.Controllers
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc

Namespace AngularReportingLocalizationExample.Controllers
	Public Class WebDocumentViewerController
		Inherits WebDocumentViewerApiController

		' GET: WebDocumentViewerApi
		Public Function Index() As ActionResult
			Return View()
		End Function
		Public Overrides Function Invoke() As ActionResult
			Dim result = MyBase.Invoke()
			Response.AppendHeader("Access-Control-Allow-Origin", "*")
			Return result
		End Function
	End Class
End Namespace